### PR TITLE
Link to build from CLI output

### PR DESCRIPTION
**Pull request by awilkes**
```
awilkes@MacBook-Pro-2 cli % python3 -m launchable record build --name foo03172
Launchable recorded 2 commits from repository /Users/awilkes/code/launchable/cli
Launchable recorded build foo03172 to workspace launchableinc/awilkes with commits from 1 repository:

| Name   | Path   | HEAD Commit                              |
|--------|--------|------------------------------------------|
| .      | .      | 6ce5dd86412c9c3010803cedeaad21b248143708 |

Visit https://app.launchableinc.com/organizations/launchableinc/workspaces/awilkes/data/builds/1647218 to view this build and its test sessions
```

### CODE REVIEW

**Comment by kohsuke** [comment]
I was surprised that this requires another round-trip. It feels like an oversight in the API design that the act of creating a build doesn't return the ID.

**Comment by awilkes** [comment]
Oh, yes, I forgot about that! Good catch.

**Review by Konboi — COMMENTED** [review]
> I was surprised that this requires another round-trip. It feels like an oversight in the API design that the act of creating a build doesn't return the ID.

Ah, you're right

I'll take over this PR and fix API side too.

File: tests/commands/record/test_build.py [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
 
 import responses  # type: ignore
 
-from launchable.utils.session import SESSION_DIR_KEY, clean_session_files, read_build
+from launchable.utils.session import read_build
 from tests.cli_test_case import CliTestCase
 
 
 class BuildTest(CliTestCase):
-    def setUp(self):
```
Comment (Konboi) [inline tests/commands/record/test_build.py:14]: It's already defined in CliTestCase.

https://github.com/launchableinc/cli/blob/7c61265e748f4551e937a295878c427255931434/tests/cli_test_case.py#L31-L33

File: tests/commands/record/test_build.py [hunk 2]
```diff
@@ -1,25 +1,14 @@
...
-        self.dir = tempfile.mkdtemp()
-        os.environ[SESSION_DIR_KEY] = self.dir
-
-    def tearDown(self):
```
Comment (Konboi) [inline tests/commands/record/test_build.py:18]: Already defined too.
https://github.com/launchableinc/cli/blob/7c61265e748f4551e937a295878c427255931434/tests/cli_test_case.py#L173-L176

**Comment by awilkes** [comment]
Thanks @Konboi!

**Review by shibuiwilliam — APPROVED** [review]

**Pull request merged**